### DISCUSSION TITLE
What does "wid" stand for?

### DISCUSSION THREAD

**Discussion by kurt-rhee** [pvlib/pvlib-python#2137] [Q&A] [answered]
Hey @kevinsa5 or @mikofski I was just curious what "wid" stands for here?  Thanks!

https://pvlib-python.readthedocs.io/en/stable/_modules/pvlib/tracking.html#singleaxis

```
wid = shading.projected_solar_zenith_angle(
    axis_tilt=axis_tilt,
    axis_azimuth=axis_azimuth,
    solar_zenith=apparent_zenith,
    solar_azimuth=apparent_azimuth,
)
```

**Comment by kandersolar** [comment, marked as answer]
It's the plaintext version of $\omega_{\text{ID}}$, the symbol for the "ideal tracking angle" in [Lorenzo et al 2011](https://doi.org/10.1002/pip.1085), the original reference for that function.  Perhaps it's worth a PR to clarify/improve those variable names?
